fuse_doc/doc/index.rst

```diff
@@ -0,0 +1,34 @@
+.. _index:
+
+fuse wiki
+*********
+
+The fuse package brings a modern graph-based approach to ROS sensor fusion, providing a flexible and extensible solution to state estimation. fuse allows users to develop custom sensor and motion models, add additional state variables as needed for tracking visual landmarks or sensor calibration parameters, and control the length of the state history anywhere from just the most recent states to maintaining the full state history for SLAM-like applications.
+
+
+.. toctree::
+   :hidden:
+
+   getting_started
+   CHANGELOG
+
+Features
+========
+
+* The state vector is customizable. In fact, there is no single state vector. Measurements can add additional state as needed. This means that vision-based landmarks can be added to the state as they are observed. Or that sensor-specific calibration variables (e.g. an IMU bias) can be added and estimated along with the robot state.
+* The robot state at multiple time points is available to sensor models. This means a sensor that measures a state change (e.g. scan-to-scan matches) can represent that as a measurement on the difference between two states.
+* Nonlinear measurement models are linearized multiple times. Ceres Solver is an iterative nonlinear solver that will linearize each measurement during each iteration. It also allows past measurements to be relinearized as new measurements are added and the state estimate changes. This reduces linearization errors in highly nonlinear measurement models.
+* The length of state history can be controlled. This can range anywhere from maintaining the previous and current state, to maintaining every state added to the system ever. This allows a single framework to be used for both live state estimation on a resource-constrained robot, to a full SLAM mapping system.
+* The motion models are a plugin-based. If an adequate motion model is not available, the user is able to write their own models for the specifics of their robot.
```

Comment:
```suggestion
* The motion models are plugin-based. If an adequate motion model is not available, the user is able to write their own models for the specifics of their robot.
```

Reply:
Thanks. That was meant to say "all" and wasn't deleted all the way.